medmodels/treatment_effect/builder.py

```diff
@@ -1,3 +1,9 @@
+"""This module contains the TreatmentEffectBuilder class.
+
+The TreatmentEffectBuilder class is used to build a TreatmentEffect object with the
+desired configurations for the treatment effect estimation using a builder pattern.
```

Comment:
imo this information should be under the actual `TreatmentEffectBuilder` and not the module description